Formula/dlib.rb

```diff
@@ -28,6 +32,16 @@ def install
     args = std_cmake_args + %w[-DDLIB_USE_BLAS=ON -DDLIB_USE_LAPACK=ON]
     args << "-DDLIB_NO_GUI_SUPPORT=ON" if build.without? "x11"
 
+    if build.with? "avx"
+      args << "-DUSE_AVX_INSTRUCTIONS=ON"
```

Comment:
Please do not create new options. If these are not automatically detected by the build process, you can use `Hardware::CPU.sse4` and `Hardware::CPU.avx`, protected by `!build.bottle?` (because prebuilt bottles need to work on all possible archs).

See `opencv` and `opencv@2` formulas for examples of how these work.